config/sites/brand.uiowa.edu/config_split.patch.administerusersbyrole.settings.yml

```diff
@@ -1,4 +1,4 @@
 adding:
   roles:
-    collaborator: safe
+    collaborator: unsafe
```

Comment:
@richardbporter does this look correct to you?

Reply:
I don't think so based on what is in config/default.